docs/release/trg-4/trg-4-08.md

```diff
@@ -0,0 +1,24 @@
+---
+title: TRG 4.08 - Multi-platform images
+---
+
+| Status | Created     | Post-History                              |
+|--------|-------------|-------------------------------------------|
+| Active | 01-Sep-2024 | Initial contribution                      |
+| Draft  | 20-Mar-2024 | Initial contribution                      |
+
+## Why
+
+Providing container images that support multiple CPU architectures enhances flexibility, compatibility, and efficiency across hardware devices.
+
+This is especially beneficial for supporting use cases like tutorials and local playgrounds.
```

Comment:
You could be a little bit more specific like mentioning ARM usage on MacOS and other runtimes like Amazon Gravition